w09h03/test/pgdp/filter/CropTest.java

```diff
@@ -0,0 +1,63 @@
+package pgdp.filter;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.awt.image.BufferedImage;
+import java.util.function.Function;
+
+public class CropTest {
+  @Test
+  void testCropExactSize() {
```

Comment:
Nice test! That was not covered yet.

Reply:
Maybe we should check for pixel equality instead